src/tests/algorithms_test/pid_MergeParticleID.cc

```diff
@@ -125,6 +125,9 @@ TEST_CASE("the PID MergeParticleID algorithm runs", "[MergeParticleID]") {
         return obj;
     }
     FAIL("ERROR: cannot find CherenkovParticleID given track");
+    if(coll->size() == 0)
```

Comment:
How nitpicky should I be on code that should never run?
```suggestion
    if(coll->empty())
```